src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -304,6 +304,10 @@ Apply to thead to make headers not bold.
     }
 }
 
+.table-column-checkbox {
+    float: none !important;
+}
+
```

Comment:
The "Main Content" area may fit better; this name and usage appears too specific for "Atomic Classes".
Also, does this rule really cause any difference in appearance? Can you include screenshots with and without it?
The `align-center` class alone may suffice, I believe.